contributor/contributors-guide.md

```diff
@@ -22,6 +46,34 @@ The docs below are for committers but may be of interest to contributors as well
 - [Verifying a Release][verifying]
 - [Testing a Release][testing]
 
+## Code Editors
+
+Feel free to use any editor when contributing Accumulo. If you are looking for a recommendation, many Accumulo
+developers use [IntelliJ] or [Eclipse]. Below are some basic instructions to help you get started.
+
+### IntelliJ
+
+1. Download and install [IntelliJ]
+1. Clone the Accumulo repository that you want to work on.
+   ```shell
+   git clone https://github.com/apache/accumulo.git
+   ```
+1. [Import][intellij-import] the repository as a Maven project into IntelliJ
+1. (Optional) Download and import `Eclipse-Accumulo-Codestyle.xml` from Accumulo's [contrib][accumulo-contrib] directory
+  * Import via `File` > `Settings` > `Code Style` and clicking on cog wheel
+
+### Eclipse
+
+1. Download and install [Eclipse].
+1. Clone the Accumulo repository that you want to work on.
+   ```shell
+   git clone https://github.com/apache/accumulo.git
+   ```
+1. [Import][eclipse-import] the repository as a Maven project into Eclipse
+1. (Optional) Download and import Eclipse formatting and style guides from Accumulo's [contrib][accumulo-contrib] directory
+  * Import Formatter: `Preferences` > `Java` > `Code Style` > `Formatter` and import the `Eclipse-Accumulo-Codestyle.xml` downloaded in the previous step.
+  * Import Template: `Preferences` > `Java` > `Code Style` > `Code Templates` and import the `Eclipse-Accumulo-Template.xml`. Make sure to check the "Automatically add comments" box. This template adds the ASF header and so on for new code.
+
```

Comment:
I don't think we need any of this.

The formatter-maven-plugin and impsort-maven-plugin take care of the formatting, for the most part, and we could always use license-maven-plugin to automatically add the license headers. We don't need to bloat the contributor docs with any of this, since none of it is important.

Perhaps the whole section should be replaced with something like:

*Use any IDE you are comfortable with. Running `mvn package -DskipTests` will format your code according to our code style. It is recommended to run this before submitting a pull request.*

Reply:
Then we should drop the files too: https://github.com/apache/accumulo/tree/master/contrib

Reply:
The files are used by the automated tooling. At least, the Codestyle one is. The Template one is only used for automatically adding a license header, but that can be automated with license-maven-plugin, and in any case, most people are probably okay with just copying the header from another source file. I'd be in favor of deleting the Template one.

Reply:
https://github.com/apache/accumulo/pull/1433 implements the license-maven-plugin I mentioned.